server/controllers/finance/reports/priceList/index.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @overview server/controllers/finance/reports/priceList/index.js
+ *
+ * @description
+ * This file contains code to create a PDF report for a price list
+ *
+ * @requires db
+ * @requires lodash
+ * @requires util
+ * @requires ReportManager
+ */
+const _ = require('lodash');
+const ReportManager = require('../../../../lib/ReportManager');
+const db = require('../../../../lib/db');
+const util = require('../../../../lib/util');
+
+
+const TEMPLATE = './server/controllers/finance/reports/priceList/report.handlebars';
+
+const PDF_OPTIONS = {
+  filename : 'FORM.LABELS.PRICE_LIST',
+};
+
+exports.report = (req, res, next) => {
+  const options = req.query;
+
+  let report;
+
+  _.defaults(options, PDF_OPTIONS);
+
+  // set up the report with report manager
+  try {
+    report = new ReportManager(TEMPLATE, req.session, options);
+
+    const priceListUUID = db.bid(req.params.uuid);
+    return lookupPriceList(priceListUUID).then(data => {
```

Comment:
Having a `return` statement within a `try {} catch{}` is a bit confusing.  What code are you afraid will error?  Is there any way to isolate the code in a `try...catch` and then continue with the rest of the code outside of the `try...catch` blocks?